common/lib/capa/capa/templates/vsepr_input.html

```diff
@@ -21,13 +21,13 @@
    />
 
       <p class="status" aria-describedby="input_${id}">
-      ${status.display_name}
+         <span class="sr">${status.display_name}</span>
       </p>
 
   <p id="answer_${id}" class="answer"></p>
 
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
Talked to @sstack22, and we don't need to worry about vsepr_input. It has at most 3 usages, and we would like to delete it once we verify it won't break running courses.